src/third-party.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: page
+permalink: /third-party/
+title: Third-party projects 
+is_homepage: false
+is_sphinx_doc: false
+---
+
+ Third-party projects
+---------
+
+### Cassandra as-a-Service cloud offerings
+
+* [Aiven for Apache Cassandra](https://aiven.io/cassandra): Aiven for Apache Cassandra is a fully managed NoSQL database, deployable in the cloud of your choice.
+Snap it into your existing workflows with the click of a button, automate away the mundane tasks, and focus on building your core apps. Now running Cassandra 3.11.
+* [Amazon Keyspaces (for Apache Cassandra)](https://aws.amazon.com/keyspaces/): Scalable, highly available, and managed Apache Cassandra–compatible database service.
+* [DataStax Astra](https://astra.datastax.com): Cloud-native database as-a-service built on Apache Cassandra™ complete with a free-tier and CQL, REST and GraphQL APIs for faster development. Deployable in AWS and GCP.
+* [Instaclustr Hosted & Managed Apache Cassandra as a Service](https://www.instaclustr.com/solutions/managed-apache-cassandra): Instaclustr provides a fully managed and SOC 2 certified hosted & managed service for Apache Cassandra® on AWS, Azure, GCP and IBM Cloud.
+
+### Cassandra installation tools
+* [Docker community Cassandra images](https://hub.docker.com/_/cassandra): Docker images for Apache Cassandra maintained by the Docker community
+* [DataStax Desktop](https://downloads.datastax.com/#desktop): Cross-platform (Windows, MacOSX, Linux) application that allows developers to quickly explore Apache Cassandra™ with a few clicks on their laptop, complete with tutorials and walkthroughs.
+* [The Last Pickle/DataStax tlp-cluster](https://github.com/thelastpickle/tlp-cluster): tlp-cluster, a tool for launching Cassandra clusters in AWS
```

Comment:
Can we just keep "The Last Pickle" as the name? And at the end of the sentence just put "(DataStax)" in?
e.g.
```suggestion
* [The Last Pickle tlp-cluster](https://github.com/thelastpickle/tlp-cluster): tlp-cluster, a tool for launching Cassandra clusters in AWS. (DataStax)
```

This applies to all the occurrences of `The Last Pickle/DataStax`

Reply:
Reasoning behind that?

Reply:
Only that it reads odd 🤷 
These tools were from The Last Pickle, and that's what the _tlp_ initials are.